### DISCUSSION TITLE
HCaptcha Error 500 / Request Validation Failure

### DISCUSSION THREAD

**Discussion by SAM-AEL** [supabase/supabase#9507] [Questions] [answered]
**Using SupabaseJS with NuxtJS.

Login and Signup works with HCaptcha disabled, but with it enabled, and passed token to the request.**

`<VueHcaptcha :sitekey="CaptchaApiToken" ref="captchaRef" @verify="setCaptchaKeys"></VueHcaptcha>`

and is passed onto signup function as

```
await supabase.auth.signInWithPassword({
  email: email.value,
  password: password.value,
  options: { captchaToken: captchaToken.value },
})
```

but with captcha enabled, it throws this error, anyone has it working?

```
{
	"code": 500,
	"msg": "request validation failure",
	"error_id": "*****- *****-*****"
}
```

**Comment by SAM-AEL** [comment, marked as answer]
My bad, it wasnt related to Supabase but rather some issue with my secret key from one account. Kindly delete this. Thank you

**Reply by peachycutiee** [reply]
i have the same problem, but my environment variable keys are correct tho..
